src/components/plots/TrajectoryAnalysisPlot.jsx

```diff
@@ -308,15 +306,8 @@ TrajectoryAnalysisPlot.propTypes = {
     PropTypes.bool,
     PropTypes.object,
   ]),
-  onUpdate: PropTypes.func.isRequired,
```

Comment:
What was this used for before? Why isn't it necessary anymore?

Reply:
It was used to dispatch `updatePlotConfig` using the `updateEmbeddingAxesIfNecessary`, which was removed so its not needed anymore